haas/test_common.py

```diff
@@ -59,6 +60,166 @@ def config_testsuite():
         })
 
 
+def initial_db():
```

Comment:
Quoting henn (2016-04-05 01:00:12)

> > +def initial_db():
>
>    Maybe move this function into a separate file, or at the very least to
>    the bottom of this file?

Moved it to the bottom of the file. `test_common` has gradually been
getting more cluttered, and we probably ought to do some general
refactoring. I'll open a separate issue for that.